src/main/java/uk/gov/ons/ctp/response/action/client/CollectionExerciseClientService.java

```diff
@@ -52,7 +52,7 @@ public CollectionExerciseDTO getCollectionExercise(final UUID collectionExcercis
 
     final HttpEntity<?> httpEntity = restUtility.createHttpEntity(null);
 
-    log.with("collection_exericse_id", collectionExcerciseId)
+    log.with("collection_exercise_id", collectionExcerciseId)
```

Comment:
Minor typo: collectionExcerciseId -> collectionExerciseId. Only matters for future searches for the name.